### ISSUE TITLE
Unreal Tournament '99 crashes (13240)

### ISSUE THREAD

**Issue by Pavuucek** [ValveSoftware/Proton#568]
I've tested this one and i sort of works.
First, don't install Dirext-X 9 pre-requisity, then run the game. It may or may not crash.
I can't actually see what the game complains about :-(
![snimek z 2018-08-24 12-03-39](https://user-images.githubusercontent.com/1253845/44620608-a2396a80-a897-11e8-8637-a85911a347ce.png)
However if you try sufficient number of times you'll get the game running eventually.
Then the game will crash everytime you change screen resolution. Next time you run it will use the resolution you set and plays fine. I've played few matches and no problems at all...

Also: why does this one use steamplay at all? There is a native linux version of this game and it works just fine without crashes.
https://wiki.dotslashplay.it/en/games/unreal-tournament

**Comment by Salamek** [comment]
@Pavuucek Yeah why is there only windows version ? UT 99 and 2004 have linux native versions...

**Comment by LeJimster** [comment]
I'm pretty sure UT99 have an OpenGL renderer option as well.   But correct there are native Linux versions of Unreal/Unreal RTNP and UT.

**Comment by adrianlshaw** [comment]
@Pavuucek your bug also happens with Deus Ex 1

**Comment by Pavuucek** [comment]
I've tried running the game with opengl renderer and it doesn't crash. Either on startup or resolution change. However after resolution change mouse returns to upper left corner of the screen everytime I move it and confirming resolution change with enter doesn't work everytime.

**Comment by martynhare** [comment]
@kisak-valve This game has a native Linux version that I can confirm works on modern distros with a bit of tweaking (namely, using paoss).  Is it possible that Valve could maybe extend Proton to allow launching native Linux versions of games where it makes clear sense to?  The Loki Games ports did ship on some of the CD releases of UT games.

**Comment by jomarocas** [comment]
for me working good, i dont know how install the latest patch but working very good, only with play button

**Comment by kisak-valve** [comment]
Can't Alt+Tab on Unreal Tournament: Game of the Year Edition (UT99)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/2187.
@absorber posted on 2019-01-03T10:43:17:

# Compatibility Report
- Name of the game with compatibility issues: Unreal Tournament: Game of the Year Edition
- Steam AppID of the game: 13240

## System Information
- GPU: Intel HD3000
- Driver/LLVM version: 3.0 Mesa 18.0.5
[steam-13240.log](https://github.com/ValveSoftware/Proton/files/2723304/steam-13240.log)

- Kernel version: 4.15.0-42-generic
- Link to full system information report as [Gist](https://gist.github.com/absorber/abe615aa12514c517de7109576e63968):
- Proton version: 3.16-4

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.



## Symptoms 
When I start the game, I can't Alt+Tab out of it anymore. This does not happen on Software Rendering.

## Reproduction
0. Start game
1. (If launch menu pops up choose Direct3D under rendering options)
3. Start a practice session
4. Alt+Tab

**Comment by xDShot** [comment]
Doesn't work on Proton 4.11-9. When it tries to create 3d window, this appears:

![image](https://user-images.githubusercontent.com/4261659/70006768-6784ae80-157f-11ea-9213-d0acd1eddb76.png)

**Comment by xDShot** [comment]
nvm, it's http://kentie.net/article/d3d11drv/index.htm causing crash

**Comment by shoober420** [comment]
# Compatibility Report
- Name of the game with compatibility issues: Unreal Tournament: Game of the Year Edition
- Steam AppID of the game: https://steamdb.info/app/13240/

## System Information
- GPU: GTX 760
- Driver/LLVM version: NVIDIA 440.82
- Kernel version: 5.6.7-zen
- https://github.com/shoober420/linux-scripts/blob/main/home/shoober420/sysinfonvidia
- Proton version: 5.6-GE-2

ORIGINAL D3D RENDER DEFAULT LOG:
[steam-13240def.log](https://github.com/ValveSoftware/Proton/files/4536451/steam-13240def.log)

ORIGINAL D3D RENDER TWEAKED INI LOG:
[steam-13240tdef.log](https://github.com/ValveSoftware/Proton/files/4536453/steam-13240tdef.log)

UPDATED OGL LOG:
[steam-13240.log](https://github.com/ValveSoftware/Proton/files/4536455/steam-13240.log)

## Symptoms
On first launch, the game will crash because the default "640x480x16" resolution and bitdepth settings are too low apparently. When going in the "UnrealTournament.ini" file and changing the resolution settings to something more modern (2048x1536x32), the game launches successfully.

Theres many updated renders you can use, but I prefer OGL (https://www.cwdohnal.com/utglr/). Also be sure to check out OldUnreal!

www.oldunreal.com

Same behavior as Unreal Gold.
https://github.com/ValveSoftware/Proton/issues/3805

**Comment by bluestang2006** [comment]
A neat little discovery, in order to get the D3D11 renderer to work I had to add the native `d3dcompiler_43.dll` and `d3ddx11_43.dll` files via winetricks to the UT99 prefix.
There are some rendering artifacts but it is a start nonetheless.
